modules/custom/az_core/src/Utility/AZBootstrapMarkupConverter.php

```diff
@@ -327,6 +327,84 @@ public static function compareProcessor($text) {
     'bg-gray-700' => 'text-bg-gray-700',
     'bg-gray-800' => 'text-bg-gray-800',
     'bg-gray-900' => 'text-bg-gray-900',
+    /***************************************************************************
+     * UA Bootstrap classes deprecated in Arizona Bootstrap 2.
+     */
+    // >> Buttons
+    'btn-default' => 'btn-red',
+    'btn-primary' => 'btn-blue',
+    'btn-hollow' => 'btn-outline',
+    'btn-hollow-default' => 'btn-outline-red',
+    'btn-hollow-primary' => 'btn-outline-blue',
+    'btn-hollow-reverse' => 'btn-outline-white',
+    // >> Labels
+    'label' => 'badge',
+    'label-default' => 'text-bg-red',
+    'label-primary' => 'text-bg-blue',
+    'label-success' => 'text-bg-success',
+    'label-info' => 'text-bg-info',
+    'label-warning' => 'text-bg-warning',
+    'label-danger' => 'text-bg-danger',
+    'label-muted' => 'text-bg-light',
+    // >> Pull Left/Right
+    'pull-left' => 'float-start',
+    'pull-right' => 'float-end',
+    // >> Colors
+    // >>> Background Colors
+    'bg-silver-tint' => 'text-bg-gray-100',
+    // >>> Card Colors
+    'card-primary' => 'text-bg-blue',
+    'card-success' => 'text-bg-success',
+    'card-info' => 'text-bg-info',
+    'card-warning' => 'text-bg-warning',
+    'card-danger' => 'text-bg-danger',
+    'card-inverse' => 'text-bg-dark text-white',
+    'card-primary-outline' => 'border-blue',
+    'card-success-outline' => 'border-success',
+    'card-info-outline' => 'border-info',
+    'card-warning-outline' => 'border-warning',
+    'card-danger-outline' => 'border-danger',
+    // >>> Text Colors
+    'text-primary' => 'text-blue',
+    // >> Navbar Default
+    'navbar-default' => 'navbar-light',
+    // >> Navbar Divider
+    'divider' => 'dropdown-divider',
+    'nav-divider' => 'dropdown-divider',
+    // >> Text
+    'bold' => 'fw-bold',
+    // >>> Page Header
```

Comment:
Adding .triangle per https://github.com/az-digital/arizona-bootstrap/blob/2.x/scss/_deprecated.scss#L370.

```suggestion
    // >> Triangles List
    'triangle' => 'az-list-triangles',
    // >>> Page Header
```